### PR TITLE
extensible loading of demo jobs

### PR DESCRIPTION
**Pull request by ClaudiaPHI**
Removed the commercial jobs from OSS and added a extensible loading of a sample jobs.

In order to test this delete the home folder in ".datacleaner" , eg. 5.1.4, and any previous folder versions.

### CODE REVIEW

**Comment by LosD** [comment]
This seems to be need changes to the users home folder, if I read you comment correct. That's unfortunately not going to fly.

**Comment by LosD** [comment]
Oh! Do you just mean for checking that they are not copied by the installer. That's of course fine

**Comment by ClaudiaPHI** [comment]
YES!

File: testware/src/main/java/org/datacleaner/cli/JobTestHelper.java [hunk 1]
```diff
@@ -76,9 +77,12 @@ public static void testJob(final String jobFileName, final Map<String, String[]>
         }
     }
 
-    private static String runJob(final String jobFileName) throws Exception {
-        final ProcessBuilder builder = new ProcessBuilder(JAVA_EXECUTABLE, "-cp", System.getProperty("java.class.path"),
-                Main.class.getCanonicalName(), "-job", jobFileName);
+    private static String runJob(final String jobFileName, final String... extraCLIArgs) throws Exception {
+        String[] processBuilderArguments = ArrayUtils.addAll(new String[] { JAVA_EXECUTABLE, DATACLEANER_MAIN_CLASS_NAME,
```
Comment (jhorcicka) [inline testware/src/main/java/org/datacleaner/cli/JobTestHelper.java]: Missing final.

File: desktop/ui/src/main/java/org/datacleaner/user/DemoConfiguration.java [hunk 2]
```diff
@@ -72,16 +60,17 @@ public static boolean isUnconfiguredDemoDatastore(Datastore ds) {
         return false;
     }
 
-    private static String addFile(String filePath) {
+    public static String addFile(String filePath) {
```
Comment (jhorcicka) [inline desktop/ui/src/main/java/org/datacleaner/user/DemoConfiguration.java]: Final argument?

**Comment by jhorcicka** [comment]
Tested and apart from one (two) missing final (@ClaudiaPHI) LGTM.

**Pull request merged**